.env

```diff
@@ -10,7 +10,14 @@ STELLIO_AUTHENTICATION_ENABLED=false
 
 ENVIRONMENT=docker
 
-APPLICATION_ENTITY_SERVICE_URL=http://search-service:8083
+# Used by subscription service when searching entities for recurring subscriptions
+# (those defined with a timeInterval parameter)
+SUBSCRIPTION_ENTITY_SERVICE_URL=http://search-service:8083
 
 # Used as a base URL by subscription service when serving contexts for notifications
-APPLICATION_STELLIO_URL=http://localhost:8080
+SUBSCRIPTION_STELLIO_URL=http://localhost:8080
+
+# Sample default configuration for the default tenant
+APPLICATION_TENANTS_0_ISSUER=https://sso.eglobalmark.com/auth/realms/stellio
```

Comment:
maybe specified `issuer` is only used when authentification is enabled ?

Reply:
indeed, added.